### ISSUE TITLE
Chrome: script can't get style sheet cuz chrome thinks it came from a different domain ?

### ISSUE THREAD

**Issue by Owyn** [Tampermonkey/tampermonkey#1520]
## Expected Behavior
Can access CSSStyleSheet of Styles added via a userscript and document.styleSheets has styles

## Actual Behavior
Can't access CSSStyleSheet of Styles added via a userscript and document.styleSheets is null as well

## Specifications

![image](https://user-images.githubusercontent.com/1309656/172464774-09693122-0be8-44da-86ce-90c30040f4fc.png)
![image](https://user-images.githubusercontent.com/1309656/172465879-3ea2c144-a672-4feb-b8a9-5e0886c9f498.png)

Tried GM_addstyle, appendchild, innerHTML ="..." to add the style, but the result is always the same - its `sheet` is null in Chrome (but works fine in FireFox)

I could probably just remove and readd back the whole style every time I want to change a param in it but is there no way to make it work again for changing single style attributes?

- Chromium: 102.0.5005.63 32bit
- TM:  4.16.1
- OS: Win 10 

## Script
https://github.com/Owyn/Center_Image/blob/master/CenterImage.user.js

Url: https://avatars.githubusercontent.com/u/10722164?s=120&v=4

**Comment by raszpl** [comment]
is this the reason GM_addStyle stopped working after todays update to [v4.18.6163]?

**Comment by derjanb** [comment]
> is this the reason GM_addStyle stopped working after todays update to [v4.18.6163]?

No, the style was added by an extension context and this is the "cross-origin" that prevents access to some properties.

GM_addStyle stopped working? Please create a new issue and add some details. Thanks.

**Comment by Owyn** [comment]
Wanted to add that `document.body.appendChild(theStyle);` doesn't seem to work now either due to security in Chrome
![image](https://user-images.githubusercontent.com/1309656/204025061-667fd225-5833-4773-a9fc-4d6cb24a17ef.png)

but luckily GM_Addstyle still works for some reason (no idea why?)
